monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+# Guía de Inicio Rápido del Monedero Monero
+*04/22/19*  
+Almutasim  
+_**Los monederos implican una de las decisiones de seguridad más importantes que toman los usuarios.**_
+
+
+Un software de un monedero de Monero es un programa para almacenar, enviar y recibir Monero. Te da control total sobre tus Monero con privacidad y sin intermediarios. Nadie puede cambiar o saber todos los detalles de tus transacciones, - a menos que eligas compartir tus claves de visualización para auditoría - poder esencial de la experienca de Monero.
+
+Para experimentar este poder, necesitarás elegir un monedero. El primer aspecto crítico de cualquier monedero es su confiabilidad. Sólo monederos recomendados por la comunidad (y el sitio [getmonero.org](https://getmonero.org/es/)) son mencionados en este artículo. A continuación, querrás elegir un monedero que se ejecute en tu dispositivo de cómputo preferido — iPhone, Android, ordenador Portátil o de Escritorio. Finalmente, deberías elegir un monedero que concuerde con tus necesidades de privacidad. Este artículo te guiará en ello.
+
+Algunos monederos operan usando su propia copia de la blockchain de Monero. A esto también se le llama ejecutar un nodo completo. Te da control sobre la puntualidad de tus transacciones. Aunque puedes confiar en un nodo remoto sin el riesgo añadido de perder tus Monero, si dependes en la disponibilidad del nodo remoto — siempre podrás recibir, pero es posible que no seas capaz de enviar. Otro beneficio de ejecutar tu propio nodo completo es el orgullo de contribuir a la red de Monero. Nodos completos honestos hacen más fuerte a Monero. Aprende más aquí: [monero.how/how-to-run-monero-node (en inglés)](https://www.monero.how/how-to-run-monero-node).
+
+Los beneficios de confiar, en su lugar, en nodos remotos — usando lo que se conoce como monederos ligeros, o algunas veces livianos — incluyen que no necesitas recursos de almacenamiento para la blockchain de Monero. La (no reducible) blockchain es más grande que 50 GB y en aumento. Algunos monederos que utilizan nodos remotos completos también usan menos poder de procesamiento y banda ancha de red. Si estos recursos son limitados en tu dispositivo de computación, un monedero ligero puede ser preferido. Este casi siempre será el caso para un monedero ejecutándose en un teléfono inteligente.
```

Comment:
I think we missed out here :see_no_evil::

"Si estos recursos son limitados en tu dispositivo de **computo**, un monedero ligero puede ser preferido."